docs/concepts/models.md

```diff
@@ -14,17 +14,39 @@ You can find more discussion of this in the [Dataclasses](dataclasses.md) sectio
 Untrusted data can be passed to a model and, after parsing and validation, Pydantic guarantees that the fields
 of the resultant model instance will conform to the field types defined on the model.
 
-!!! note
-    Pydantic is primarily a parsing and transformation library, **not a validation library**.
-    Validation is a means to an end: building a model which conforms to the types and constraints provided.
+!!! note "Validation — a _deliberate_ misnomer"
+    ### TL;DR
+
+    We use the term "validation" to refer to the process of constructing a model that adheres to specified types and
+    constraints. This task, which Pydantic is well known for, is most widely recognized as "validation" in colloquial terms,
+    even though in other contexts the term "validation" may be more restrictive.
+
+    ---
+
+    ### The long version
+
+    The potential confusion around the term "validation" arises from the fact that, strictly speaking, Pydantic's
+    primary focus doesn't align precisely with the dictionary definition of "validation":
+
+    > ### validation
+    > _noun_
+    > the action of checking or proving the validity or accuracy of something.
+
+    In Pydantic, the term "validation" refers to the process of constructing a model that adheres to specified
```

Comment:
```suggestion
    In Pydantic, the term "validation" refers to the process of instantiate a model (or other type) that adheres to specified
```

Reply:
Good call. Changed to "instantiating a model (or other type)" both here and in the TL;DR section.